WordPress/src/test/java/org/wordpress/android/ui/jetpack/backup/BackupDownloadDetailsViewModelTest.kt

```diff
@@ -0,0 +1,73 @@
+package org.wordpress.android.ui.jetpack.backup
+
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.TEST_DISPATCHER
+import org.wordpress.android.test
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider
+import org.wordpress.android.ui.jetpack.backup.details.BackupDownloadDetailsViewModel
+import org.wordpress.android.ui.jetpack.backup.details.BackupDownloadDetailsViewModel.UiState
+import org.wordpress.android.ui.jetpack.backup.details.BackupDownloadDetailsViewModel.UiState.Content
+
+@InternalCoroutinesApi
+class BackupDownloadDetailsViewModelTest : BaseUnitTest() {
+    private lateinit var viewModel: BackupDownloadDetailsViewModel
+    private lateinit var availableItemsProvider: BackupAvailableItemsProvider
+
+    @Before
+    fun setUp() {
+        availableItemsProvider = BackupAvailableItemsProvider()
+        viewModel = BackupDownloadDetailsViewModel(
+                availableItemsProvider,
+                TEST_DISPATCHER,
+                TEST_DISPATCHER
+        )
+    }
+
+    @Test
+    fun `when available items are fetched, the content view is shown`() = test {
+        val uiStates = initObservers().uiStates
+
+        viewModel.start()
+
+        assertThat(uiStates[0]).isInstanceOf(UiState.Content::class.java)
```

Comment:
Minor (🔍 ): The explicit `UiState` can be omitted, or the `UiState.Content` import can be removed and the `UiState` can be made explicit in places required.